lms/static/sass/shared/_header.scss

```diff
@@ -377,6 +388,19 @@ header.global-new {
 
   .nav-global {
     @include float(left);
+    .tab-nav-item{
+      .tab-nav-link{
+        height: 43px;
+        font-size: em(16);
+        color: $gray;
+        &:hover{
```

Comment:
I just spoke to @marcotuts and it was agreed that the styles should match those in LMS (see the screenshot)
![screen shot 2016-04-13 at 11 49 34 am](https://cloud.githubusercontent.com/assets/2052185/14499911/0537f882-016e-11e6-9b93-82cb7d99f7ea.png)